examples/basic_bot.py

```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
     """Repeats a message multiple times."""
     for i in range(times):
         await ctx.send(content)
-
+@bot.command()
+async def ping(ctx):
+    """A simple ping command!""" 
+    embed = discord.Embed(title="Pong!", description=f"```{round(bot.latency * 1000)} ms```", color = discord.Color.blue()) 
+    await ctx.send(embed=embed)
+@bot.event 
+async def on_command_error(ctx, error):
+	error = getattr(error, 'original', error)
+	if isinstance(error, commands.MissingRequiredArgument):
```

Comment:
This eats every other error. No-go.

Reply:
Then, other handlers could be added too, in this way it'll not eat other errors.

Reply:
Consider adding an `else:` block to raise the error if it's of another type.

Reply:
okai, thx, will add...